### PR TITLE
Add C package xclip recipe

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ehfd** [comment]
Courtesy of @danielnachun. I would like to merge this into conda-forge. Thanks.

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c, ready for review!

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-c, ready for review!

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-c-cpp, ready for review!

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-c-cpp, ready for review!

**Comment by ehfd** [comment]
Ready to be merged. Windows build impossible due to fork command.

File: recipes/xclip/COPYING [hunk 1]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (xhochy) [inline recipes/xclip/COPYING:1]: It looks like this should be in the tarball. If so, please remove this from the recipe submission.

File: recipes/xclip/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+{% set version = "0.13" %}
+{% set sha256 = "ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/astrand/xclip/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/xclip/meta.yaml:11]:
```suggestion
  sha256: ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758

```

File: recipes/xclip/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{% set name = "xclip" %}
+{% set version = "0.13" %}
+{% set sha256 = "ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758" %}
```
Comment (xhochy) [inline recipes/xclip/meta.yaml:3]: No need for templating this, it is only used once.
```suggestion
```

File: recipes/xclip/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - gettext                       # [unix]
+    - libtool                       # [unix]
+  host:
+    - xorg-libx11 >=1.6,<2
+    # Additional constraintless xorg-libx11 dep below is for conda-build to add potential pinning to.
+    - xorg-libx11
+    - xorg-libxmu
+  run:
+    - {{ pin_compatible('xorg-libx11') }}
+    - {{ pin_compatible('xorg-libxmu') }}
```
Comment (xhochy) [inline recipes/xclip/meta.yaml:36]: These have run_exports and thus should be automatically added here.
```suggestion
```

File: recipes/xclip/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage(name|lower) }}
+
+requirements:
+  build:
+    - gnuconfig  # [unix]
```
Comment (xhochy) [inline recipes/xclip/meta.yaml:22]: You don't seem to use this package in the build script.

File: recipes/xclip/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: https://github.com/astrand/xclip/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: true
+  run_exports:
```
Comment (xhochy) [inline recipes/xclip/meta.yaml:17]: This package only contains an executable, not a shared library, so it shouldn't have a `run_exports`.

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by xhochy** [comment]
As with the other PR, this should be fix in `xorg-libxmu-feedstock`

**Comment by ehfd** [comment]
@xhochy Everything should be ready after https://github.com/conda-forge/xorg-libxmu-feedstock/pull/9

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@xhochy All ready. Windows build is not done because 1) it uses `fork()`, and 2) this isn't an official X11 package, so I'm not trying that hard to work around `fork()`.

**Comment by ehfd** [comment]
@conda-forge-admin, please ping conda-forge/help-c-cpp, ready for review!

**Comment by ehfd** [comment]
This is an optional dependency of (indirectly) `pandas` and (directly) `pyperclip`.

**Pull request merged**